pyshtools/shio/common.py

```diff
@@ -0,0 +1,28 @@
+"""
```

Comment:
I wonder if this function should be in utils, or if the single function in utils should be simply in shio. Not sure what the best approach is.